server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -1620,16 +1612,6 @@ public void run() {
                     TimeValue.nsecToMSec(System.nanoTime() - startTimeNS)
                 );
             }
-
-            try {
-                this.requestCache.cleanCache();
-            } catch (Exception e) {
-                logger.warn("Exception during periodic request cache cleanup:", e);
-            }
-            // Reschedule itself to run again if not closed
-            if (closed.get() == false) {
-                threadPool.scheduleUnlessShuttingDown(interval, ThreadPool.Names.SAME, this);
-            }
```

Comment:
I guess we still need to keep this for field data cache?

Reply:
Ouch, thanks for catching it !